src/main/java/bio/terra/app/usermetrics/BardClient.java

```diff
@@ -23,30 +29,46 @@ public class BardClient {
 
   private final UserMetricsConfiguration metricsConfig;
 
-  private final RestTemplate restTemplate;
+  private final RestTemplate apiRestTemplate;
+
+  private final RestTemplate syncRestTemplate;
   private final HttpHeaders headers;
 
+  private static final int BEARER_TOKEN_CACHE_TIMEOUT_SECONDS = 3600;
```

Comment:
Just to be explicit
```suggestion
  private static final int DEFAULT_BEARER_TOKEN_CACHE_TIMEOUT_SECONDS = 3600;
```

Reply:
It's a shame `ExpiringMap` doesn't accept `Duration`.